docs/config-options.md

```diff
@@ -0,0 +1,65 @@
+# Config Options
+
+This file documents all the configuration options supported by the Spanner Cassandra Adapter.
+
+```yaml
+# [Optional] Global client configurations that apply to all listeners.
+globalClientConfigs:
+  # [Optional] The Spanner endpoint to use. Defaults to "spanner.googleapis.com:443".
+  spannerEndpoint: "spanner.googleapis.com:443"
+  # [Optional] Enables built-in metrics. Defaults to false.
```

Comment:
I would also add line saying, It is highly recommended to enable metrics in prod environment for debugging any issue